core/src/main/java/org/radargun/stages/monitor/JgroupsProbeMonitorStartStage.java

```diff
@@ -0,0 +1,78 @@
+package org.radargun.stages.monitor;
+
+import java.io.IOException;
+
+import org.radargun.DistStageAck;
+import org.radargun.config.Property;
+import org.radargun.config.Stage;
+import org.radargun.stages.AbstractJGroupsProbeStage;
+import org.radargun.sysmonitor.Monitor;
+import org.radargun.sysmonitor.ProbeSlaveMonitor;
+import org.radargun.utils.TimeConverter;
+
+/**
+ *
+ * Starts collecting probe statistics on each slave node.
+ * {@link ProbeSlaveMonitor}
+ *
+ * @author Diego Lovison &lt;dlovison@redhat.com&gt;
+ */
+@Stage(name="jgroups-probe-monitor-start", doc = "Starts collecting jgroups statistics locally in each slave node.")
+public class JgroupsProbeMonitorStartStage extends AbstractJGroupsProbeStage {
+
+   @Property(doc = "Period of statistics collection. The default is 60 seconds.", converter = TimeConverter.class)
+   private long period = 60_000;
+
+   @Override
+   public DistStageAck executeOnSlave() {
+
+      if (groups == null || (slaveState.getGroupName() != null && groups.contains(slaveState.getGroupName()))) {
```

Comment:
This condition (and more) is already covered in `AbstractDistStage.shouldExecute()`